### PR TITLE
Backport43 julio microos #20162 keichwa

### PR DESCRIPTION
**Pull request by keichwa**
# Description

See https://github.com/SUSE/spacewalk/issues/20162 and https://github.com/uyuni-project/uyuni-docs/pull/2088

# Target branches

Which documentation version does this PR apply to?

- [ ] Master (Default)
- [x] Manager-4.3
- [ ] Manager-4.2

# Links

Fixes #<insert issue or PR link, if any>

### CODE REVIEW

**Review by juliogonzalez — CHANGES_REQUESTED** [review]
The idea is to backport only the removal of MicroOS.

Later, when 4.3 gets Leap Micro support, you can backport the rest, without ifdefs

File: modules/administration/pages/actions.adoc [hunk 1]
```diff
@@ -72,7 +72,7 @@ For transactional update systems, an action chain is executed inside a single sn
 This can cause some limitations.
 
 ifeval::[{uyuni-content} == true]
-For more information, see xref:client-configuration:clients-slemicro.adoc[] and xref:client-configuration:clients-microos.adoc[].
+For more information, see xref:client-configuration:clients-slemicro.adoc[] and xref:client-configuration:clients-opensuseleapmicro.adoc[].
```
Comment (keichwa) [inline modules/administration/pages/actions.adoc:75]:
```suggestion
For more information, see xref:client-configuration:clients-slemicro.adoc[] and xref:client-configuration:clients-microos.adoc[].
```

File: modules/administration/pages/actions.adoc [hunk 2]
```diff
@@ -141,7 +141,7 @@ For transactional update systems, consider that a remote command is run inside a
...
```
Comment (keichwa) [inline modules/administration/pages/actions.adoc:144]:
```suggestion
For more information, see xref:client-configuration:clients-slemicro.adoc[] and xref:client-configuration:clients-microos.adoc[].
```

File: modules/client-configuration/nav-client-configuration-guide.adoc [hunk 3]
```diff
@@ -14,7 +14,10 @@ endif::[]
 *** xref:supported-features-sles.adoc[SLES Supported Features]
 *** xref:supported-features-es.adoc[SLES-ES Supported Features]
 *** xref:supported-features-sle-micro.adoc[SLE Micro Supported Features]
-
+// openSUSE Leap Micro not available at SUSE Manager for now
+ifeval::[{uyuni-content} == true]
+*** xref:supported-features-opensuse-leap-micro.adoc[openSUSE Leap Micro Supported Features]
+endif::[]
```
Comment (keichwa) [inline modules/client-configuration/nav-client-configuration-guide.adoc:20]:
```suggestion
```

File: modules/client-configuration/nav-client-configuration-guide.adoc [hunk 4]
```diff
@@ -52,9 +55,9 @@ endif::[]
 **** xref:clients-sleses.adoc[Expanded Support Clients]
 *** xref:registration-overview-opensuse.adoc[openSUSE Client Registration]
 **** xref:clients-opensuseleap.adoc[openSUSE Leap Clients]
-
+// openSUSE Leap Micro not available at SUSE Manager for now
```
Comment (keichwa) [inline modules/client-configuration/nav-client-configuration-guide.adoc:58]:
```suggestion
```

File: modules/client-configuration/nav-client-configuration-guide.adoc [hunk 5]
```diff
@@ -52,9 +55,9 @@ endif::[]
...
 ifeval::[{uyuni-content} == true]
-**** xref:clients-microos.adoc[openSUSE MicroOS]
+**** xref:clients-opensuseleapmicro.adoc[openSUSE Leap Micro Clients]
```
Comment (keichwa) [inline modules/client-configuration/nav-client-configuration-guide.adoc:60]:
```suggestion
**** xref:clients-microos.adoc[openSUSE MicroOS]
```

**Review by keichwa — COMMENTED** [review]
then I probably must start over again.

**Comment by keichwa** [comment]
Yes, good idea.  This is the new PR: https://github.com/uyuni-project/uyuni-docs/pull/2130

**Pull request closed without merging**